x-pack/plugins/security_solution/public/management/hooks/response_actions/use_get_endpoint_action_list.ts

```diff
@@ -40,6 +40,7 @@ export const useGetEndpointActionList = (
   return useQuery<ActionListApiResponse, IHttpFetchError<ErrorType>>({
     queryKey: ['get-action-list', query],
     ...options,
+    keepPreviousData: true,
```

Comment:
I'm approving, but am wondering if this has any broader impact?

The docs say this for this option:

```
If set, any previous data will be kept when fetching new data because the query key changed.
```

Reply:
One way to test might be to expand the row on the page. Then refresh the browser tab - I think we maintain state in the URL, so on a tab refresh there should be nothing for react-query to `keepPreviousData`